client/build.gradle

```diff
@@ -85,3 +83,60 @@ dependencies {
     runtimeOnly libraries.lwjgl_yoga
     runtimeOnly libraries.lwjgl_zstd
 }
+
+// copy dependency jars to build/libs/dependency-jars
+task copyDependencies (type: Copy) {
+    def toDir = "${buildDir}/libs/dependency-jars/"
+
+    // create directories, if not already done:
+    file(toDir).mkdirs()
+
+    // copy jars to lib folder:
+    from "${buildDir}/resources/main"
+    from configurations.default
+    into toDir
+}
+
+jar {
+    // exclude log properties (recommended)
+    exclude ("log4j.properties")
+
+    // make jar executable: see http://stackoverflow.com/questions/21721119/creating-runnable-jar-with-gradle
+    manifest {
+        attributes (
+                'Main-Class': 'com.github.jsappz.lamegame.game.Main',
+                // add classpath to Manifest; see http://stackoverflow.com/questions/30087427/add-classpath-in-manifest-file-of-jar-in-gradle
+                "Class-Path": '. dependency-jars/' + configurations.default.collect { it.getName() }.join(' dependency-jars/')
+        )
+    }
+}
+
+// always call copyJarsToLib when building jars:
+jar.dependsOn copyDependencies
+
+def embeddedJRE = true
+
+task javapackager(type: Exec, dependsOn: [assemble]) {
```

Comment:
May either want a ticket to copy this to the server, or have them packaged together top level with options how which one to run? Wasn't sure, your call @JReistad

Reply:
Another potential follow up is some sort of integration test setup between the client and the server. Docker might be a good option depending on how we plan to deploy the server. Your Call @JReistad

Reply:
👍 thanks for calling out. will probably let this one stew a while until we figure out what seems fitting. Will start a long term backlog. 
We'll definitely want some integration tests between client and server at some point in time, but seems early at this point.